.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: ci
+
+on:
+  pull_request:
+  push:
+    branches: master
+
+jobs:
+  test:
+    name: test w/ Python ${{ matrix.python-version }}
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install test deps
+        run: python3 -m pip install -U coverage mypy
```

Install wheel to avoid:

```
Using legacy 'setup.py install' for cffi, since package 'wheel' is not installed.
Using legacy 'setup.py install' for multidict, since package 'wheel' is not installed.
Using legacy 'setup.py install' for yarl, since package 'wheel' is not installed.
```

```suggestion
      - name: Install test deps
        run: |
          python3 -m pip install -U wheel
          python3 -m pip install -U coverage mypy
```